2009-04-30  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Sizes.pm: Clarified error message in cases where the user
		specified a partition is larger than the entire disk
Index: trunk/lib/setup-storage/Sizes.pm
===================================================================
--- trunk.orig/lib/setup-storage/Sizes.pm
+++ trunk/lib/setup-storage/Sizes.pm	
@@ -74,7 +74,7 @@
   }
 
   # the user may have specified a partition that is larger than the entire disk
-  ($start <= $size_b) or die "Lower bound of partition size is greater than disk size\n";
+  ($start <= $size_b) or die "Sorry, can't create a partition of $start B on a disk of $size_b B - check your config!\n";
   # make sure that $end >= $start
   ($end >= $start) or &FAI::internal_error("end < start");
 
